tools/scripts/downloadMinecraftServer.sh

```diff
@@ -18,10 +18,14 @@ downloadTargettedVersion() {
     versionSpecificServerUri=$(grep -o "https:\/\/launcher\.mojang\.com\/v1\/objects\/.\{40\}\/server\.jar" .cache/$mcVersion/version_manifest.json)
     versionSpecificMappingsUri=$(grep -o "https:\/\/launcher\.mojang\.com\/v1\/objects\/.\{40\}\/server\.txt" .cache/$mcVersion/version_manifest.json)
 
+    serverProGuard=".cache/$mcVersion/server.txt"
+    serverTiny=".cache/$mcVersion/server.tiny"
     curl --create-dirs -s -o ".cache/$mcVersion/server.jar" "$versionSpecificServerUri" || exit 1
-    curl --create-dirs -s -o ".cache/$mcVersion/server.txt" "$versionSpecificMappingsUri" || exit 1
+    curl --create-dirs -s -o $serverProGuard "$versionSpecificMappingsUri" || exit 1
+    java -jar tools/enigma-cli-0.21.6+build.229-all.jar convert-mappings proguard $serverProGuard tinyv2:obf:deobf $serverTiny || exit 1
```

Comment:
This really isn't the correct place to put this conversion. It might be better suited in the decompile step or a new step between this one and the decompile.